stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/textinput/StreamMessageInputView.kt

```diff
@@ -45,10 +49,19 @@ public class StreamMessageInputView : ConstraintLayout {
     private var currentAnimatorSet: AnimatorSet? = null
 
     private lateinit var binding: StreamMessageInputBinding
+    private lateinit var suggestionListController: SuggestionListController
 
     private var iconDisabledSendButtonDrawable: Drawable? = null
     private var iconEnabledSendButtonDrawable: Drawable? = null
 
+    public fun configureMembers(members: List<Member>) {
+        suggestionListController.members = members
+    }
+
+    public fun configureCommands(commands: List<Command>) {
+        suggestionListController.commands = commands
+    }
```

Comment:
Don't we have some kind of work to perform after these values change? Do we only want to update the next time that the text input changes as well?

Reply:
Done, recalculate suggestions in controller when new values arrive